src/services/Api.ts

```diff
@@ -0,0 +1,33 @@
+import axios, { AxiosRequestConfig } from 'axios'
+import { Pokemon } from '../interfaces'
+
+interface ListConfig{
+	length: number,
+	initialValue: number
+}
+
+class Http{
+	static service = axios.create({
+		baseURL: import.meta.env.VITE_BASE_URL as string,
+	})
+	
+	static get(resource: string, config?: AxiosRequestConfig){
+		return Http.service.get(resource, config)
+	}
+	
+	static getPokemon(identificator: string | number){
+		return Http.get(`pokemon/${identificator}`)
+	}
+	
+	static async getPokemonList({ length, initialValue = 0 }: ListConfig){
+		const { data } = await Http.get(`pokemon?limit=${length}&offset=${initialValue}`)
+		const pokeList = Promise.all( await data.results.map(async (e:{name:string, url: string}) => {
```

Comment:
Oh no! Is this the only way of fetching the pokemons?
It's bad for `getPokemonList` to make `1+length` requests.
What happens if a single request fails?